server/webapp/WEB-INF/rails.new/app/assets/stylesheets/patterns/forms.scss

```diff
@@ -175,6 +175,10 @@ label.inline {
     padding-bottom:0;
 }
 
+.form_item_block.with-padding-top {
```

Comment:
Merge rule `.form_item_block.with-padding-top` with rule on line 170<br>Selector `form_item_block` should be written in lowercase with hyphens